plot-geotiff-pixel-bbox.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+###########################################################
+# Plots a GeoTiff with a pixel count bounding box overlay.
+# Use to determine pixel values for ground control points
+# when geo-referencing or creating/testing cutlines.
+#######################################################
+import argparse
+import logging
+import os
+import rasterio
+from rasterio import windows
+import matplotlib.pyplot as plt
+from matplotlib.patches import Rectangle
+
+logging.basicConfig(format='%(message)s', level=logging.INFO)
+logger = logging.getLogger()
+
+
+def run(args):
+    src_file = args.src
+    min_x = args.min_x
+    max_x = args.max_x
+    min_y = args.min_y
+    max_y = args.max_y
+
+    logging.info('Reading band 1 from ' + src_file)
+    raster_src = rasterio.open(src_file)
+
+    logging.info('Building overlay')
+    slice_raster = (slice(min_y, max_y), slice(min_x, max_x))
+    window_slice = windows.Window.from_slices(*slice_raster)
+
+    plt.imshow(raster_src.read(1))
+    plt.title(os.path.basename(src_file))
+    ax = plt.gca()
+    ax.add_patch(
+        Rectangle(
+            (window_slice.col_off, window_slice.row_off),
+            width=window_slice.width,
+            height=window_slice.height,
+            fill=True,
+            alpha=.5,
+            color="red"
```

Comment:
I think red is probably a good color choice for the majority of use-cases, but it could be convenient to have this be a parameter too (e.g. `--bbox_color purple`) in case red doesn't cut it for some rasters.